### PR TITLE
Create test for restoring a snapshot with dropped columns (CASSANDRA-13276)

### PR DESCRIPTION
**Pull request by adelapena**

### CODE REVIEW

File: snapshot_test.py [hunk 1]
```diff
@@ -106,6 +114,41 @@ def test_basic_snapshot_and_restore(self):
...
 
+    def test_snapshot_and_restore_dropping_a_column(self):
+        """
+        @jira_ticket CASSANDRA-13276
+
+        Can't load snapshots of tables with dropped columns.
+        """
+        cluster = self.cluster
+        cluster.populate(1).start()
+        (node1,) = cluster.nodelist()
```
Comment (ptnapoleon) [inline snapshot_test.py]: Tiny nit: Could you remove the `,` here? It's unnecessary.

**Comment by ptnapoleon** [comment]
+1

**Comment by adelapena** [comment]
Thanks for checking, @ptnapoleon.

Did you mean the parenthesis instead of the comma?
```
>>> l = [1]
>>> (x) = l
>>> x
[1]
>>> x, = l
>>> x
1
```

**Comment by ptnapoleon** [comment]
Ahh, my bad @adelapena, I thought parentheses worked the same way as brackets:
```
In [1]: l = [1]
In [2]: [x] = l
In [3]: x
Out[3]: 1
```
In which case yes, please remove the parentheses, and keep the comma

**Comment by adelapena** [comment]
Done

**Comment by ptnapoleon** [comment]
+1. Please squash on merge.

**Comment by adelapena** [comment]
Merged, thanks :)

**Pull request merged**